bin/meshroom_batch

```diff
@@ -140,8 +140,15 @@ with multiview.GraphModification(graph):
         graph.setVerbose(args.verbose)
 
     if args.output:
-        publish = getOnlyNodeOfType(graph, 'Publish')
```

Comment:
getOnlyNodeOfType is not used anymore

Reply:
Done, `getOnlyNodeOfType` has been completely removed.